src/flash/nor/rp2040.c

```diff
@@ -172,6 +172,23 @@ static int rp2040_flash_write(struct flash_bank *bank, const uint8_t *buffer, ui
 		return ERROR_TARGET_UNALIGNED_ACCESS;
 	}
 
+	// This is a good place to do this because flash erase always follows a reset init
```

Comment:
I would remove this comment which is misleading

Reply:
Done